server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -265,12 +297,17 @@ export function configureWebSocketServices(
 
             let connectedMessage: IConnected;
             if (isWriter(messageClient.scopes, details.existing, message.mode)) {
-                const orderer = await orderManager.getOrderer(claims.tenantId, claims.documentId);
-                const connection = await orderer.connect(socket, clientId, messageClient as IClient, details);
-                // eslint-disable-next-line @typescript-eslint/no-floating-promises
-                connection.connect();
+                const orderer = await orderManager.getOrderer(claims.tenantId, claims.documentId)
+                    .catch(async (err) => {
+                        const errMsg = `Failed to get orderer manager. Error: ${safeStringify(err, undefined, 2)}`;
```

Comment:
(nit) I think I prefer this error message format ("Failed to x. Error: ${err}") over the other ("Failed to x due to error: ${err}"). Perhaps we could make them all in this format?

Reply:
Makes sense! I updated the error messages so they should have the same format now.